license.txt

```diff
@@ -1,31 +1,24 @@
- Copyright (c) 2007-2019 Mike Karlesky, Mark VanderVoord, Greg Williams
+Copyright (c) 2011-2024 Michael Karlesky, Mark VanderVoord, Greg Williams
 
- Permission is hereby granted, free of charge, to any person
- obtaining a copy of this software and associated documentation
- files (the "Software"), to deal in the Software without
- restriction, including without limitation the rights to use,
- copy, modify, merge, publish, distribute, sublicense, and/or sell
- copies of the Software, and to permit persons to whom the
- Software is furnished to do so, subject to the following
- conditions:
+https://opensource.org/license/mit/
```

Comment:
Is this link here actually needed? Because if not, the license would be `MIT`, otherwise, i.e., including this link, the license is custom.

Reply:
Fixed in f57f423bb7aaf870343be04413a68718518a4593.